test/acceptance/test/install_tests.go

```diff
@@ -103,8 +105,22 @@ var _ = Describe("Weave GitOps Install Tests", func() {
 
 		installAndVerifyWego(namespace, appRepoRemoteURL)
 
-		By("When I run 'gitops uninstall' command", func() {
-			_ = runCommandPassThrough([]string{}, "sh", "-c", fmt.Sprintf("%s uninstall --namespace %s", WEGO_BIN_PATH, namespace))
+		By("When I run 'gitops uninstall' command without force flag it asks for confirmation", func() {
+			cmd := fmt.Sprintf("%s uninstall --namespace %s", WEGO_BIN_PATH, namespace)
+			outputStream := gbytes.NewBuffer()
+			inputUser := bytes.NewBuffer([]byte("y\n"))
+
+			c := exec.Command("sh", "-c", cmd)
+			c.Stdout = outputStream
+			c.Stdin = inputUser
+			c.Stderr = os.Stderr
+			err := c.Start()
+			Expect(err).ShouldNot(HaveOccurred())
+
+			Eventually(outputStream).Should(gbytes.Say(`Uninstall will remove all your Applications and any related cluster resources\. Are you sure you want to uninstall\? \[y\/N\]`))
+
+			err = c.Wait()
+			Expect(err).ShouldNot(HaveOccurred())
```

Comment:
@josecordaz we're not verifying any actions _after_ the user provides an input `(y/n)`. Is that on purpose?

Reply:
Actually, we are. Particularly with this test we are validating that the namespace gets removed after running the uninstall command, this can only happen if the "user" types `y` and then enter which is being emulated in this line https://github.com/weaveworks/weave-gitops/pull/1073/files#diff-93d1ec3b2fa24c2149cf72486d97a7dfae21bad613096ee91d767aa070bfe58fR111.